_config.yml

```diff
@@ -64,7 +78,7 @@ paginate: 12
 ##################################
 
 # Custom CSS file path
-custom_css_path: "/do-not-touch/theme-colors/custom.css"
+custom_css_path: "/misc/custom.css"
```

Comment:
can we double check here that our `build.sh` overrides?

Reply:
marking as resolved; `build.sh` does **not** override this but i think we create the site with this key as default